source

```
@@ -93222,45 +93242,58 @@ loadMySprites().then(runDemo);</pre>
+
+  <ol>
+   <li><p>Increment <var>target object</var>'s <span>animation frame callback identifier</span> by
    one.</p></li>
 
-   <li><p>Append the method's argument to <var>document</var>'s <span>list of animation frame
-   callbacks</span>, associated with <var>document</var>'s <span>animation frame callback
+   <li><p>Append the method's argument to <var>target object</var>'s <span>list of animation frame
+   callbacks</span>, associated with <var>target object</var>'s <span>animation frame callback
    identifier</span>'s current value.</p></li>
 
-   <li><p>Return <var>document</var>'s <span>animation frame callback identifier</span>'s current
-   value.</p></li>
+   <li><p>Return <var>target object</var>'s <span>animation frame callback identifier</span>'s
+   current value.</p></li>
 
   </ol>
 
-  <p>When the <dfn><code
-  data-x="dom-window-cancelAnimationFrame">cancelAnimationFrame()</code></dfn> method is called,
-  the user agent must run the following steps:</p>
+  <p>When the <dfn>
+  <code data-x="AnimationFrameProvider-cancelAnimationFrame">cancelAnimationFrame()</code></dfn>
+  method is called, the user agent must run the following steps:</p>
 
   <ol>
+   <li><p>Find the entry in <var>target object</var>'s <span>list of animation frame callbacks
+   </span> that is associated with the value given by the method's argument.</p></li>
 
-   <li><p>Let <var>document</var> be this <code>Window</code> object's <span
-   data-x="concept-document-window">associated <code>Document</code></span>.</p></li>
-
-   <li><p>Find the entry in <var>document</var>'s <span>list of animation frame callbacks</span>
-   that is associated with the value given by the method's argument.</p></li>
-
-   <li><p>If there is such an entry, remove it from <var>document</var>'s <span>list of animation
-   frame callbacks</span>.</p></li>
+   <li><p>If there is such an entry, remove it from <var>target object</var>'s <span>list of
```

is it possible that the entry is null?

---

it's possible that there's no entry. But current spec (and implementations) says it should fail silently when it can't find an entry.